indico/modules/events/contributions/models/fields.py

```diff
@@ -44,6 +44,10 @@ class ContributionField(db.Model):
         index=True,
         nullable=False
     )
+    legacy_id = db.Column(
+        db.String,
+        nullable=True
+    )
```

Comment:
should we already add a `friendly_id` too? (and use that to generate the legacy abstract ID instead of letting ZODB generate it)